services/userService.js

```diff
@@ -0,0 +1,69 @@
+const jwt = require('jsonwebtoken');
+const axios = require('axios');
+const userDao = require('../models/userDao');
+
+const getKakaoUserInfo = async (kakaoAccessToken) => {
+  let kakaoUserInfo;
+
+  await axios({
+    method: 'POST',
+    url: 'https://kapi.kakao.com/v2/user/me',
+    headers: {
+      'Content-type': 'application/x-www-form-urlencoded;charset=utf-8',
+      'Authorization': `Bearer ${kakaoAccessToken}`
+    }
+    }).then((result) => {
+      kakaoUserInfo = result.data;
+    })
+  
+  return kakaoUserInfo
+};
```

Comment:
async await 키워드를 사용했기 떄문에 `.then`을 사용하여 비동기처리를 하지 않아도 될 것 같습니다.

```suggestion
const getKakaoUserInfo = async (kakaoAccessToken) => {
  const options = {
    method: 'POST',
    url: 'https://kapi.kakao.com/v2/user/me',
    headers: {
      'Content-type': 'application/x-www-form-urlencoded;charset=utf-8',
      'Authorization': `Bearer ${kakaoAccessToken}`
    }}
  const kakaoUserInfo = await axios.request(options)
  
  return kakaoUserInfo
};
```

Reply:
```suggestion
const getKakaoUserInfo = async (kakaoAccessToken) => {
  let kakaoUserInfo;

  await axios({
    method: 'POST',
    url: 'https://kapi.kakao.com/v2/user/me',
    headers: {
      'Content-type': 'application/x-www-form-urlencoded;charset=utf-8',
      'Authorization': `Bearer ${kakaoAccessToken}`
    }
    }).then((result) => {
      kakaoUserInfo = result.data;
    })
  
  return kakaoUserInfo
};
```


const getKakaoUserInfo = async (kakaoAccessToken) => {

  const result = await axios({
    method: 'POST',
    url: 'https://kapi.kakao.com/v2/user/me',
    headers: {
      'Content-type': 'application/x-www-form-urlencoded;charset=utf-8',
      'Authorization': `Bearer ${kakaoAccessToken}`
    }
  })

  const kakaoUserInfo = result.data;
  
  return kakaoUserInfo
};

Reply:
테스트 결과, 문제가 없어 위 코드대로 수정하도록 하겠습니다!